app/components/dashboard/audit_information_component.html.erb

```diff
@@ -60,7 +60,7 @@ Note also:
       </tr>
       <tr>
         <td><strong>Catalog to Archive</strong><br>Confirm a PreservedObject has all versions/parts replicated for each of its target endpoints and attempts to backfill missing ones</td>
-        <td>moab_replication_audit<br>part_audit_aws_s3_east_1<br>part_audit_aws_s3_west_2<br>part_audit_ibm_us_south</td>
+        <td>moab_replication_audit</td>
```

Comment:
endpoint specific audit functionality was folded into one replication audit job during the heavy pres cat maintenance of early 2023